### ISSUE TITLE
NOT_DONE_YET is missing from __all__ in twisted.web.server

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4966]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)| @lvh reported|
|-|-|
|Trac ID|trac#4966|
|Type|defect|
|Created|2011-03-20 01:51:28Z|
|Branch|https://github.com/twisted/twisted/tree/not-done-yet-4966|

This causes Pydoctor not to document it, which causes broken links in the documentation. Similar to [#3904](https://github.com/twisted/twisted/issues/3904), found in [#4951](https://github.com/twisted/twisted/issues/4951).

Attachments:

* [twisted4966.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/2cf/2cf677a4567a1c787edd581dd898449285eff9df/053bda9ac719c4cc1e51dc05b0a3131c4019acf2.patch) (527 bytes) - added by necaris on 2012-03-12 23:13:32Z - Patch to add __all__ to twisted/web/server.py

<details><summary>Searchable metadata</summary>

```
trac-id__4966 4966
type__defect defect
reporter__lvh lvh
priority__normal normal
milestone__ 
branch__branches_not_done_yet_4966 branches/not-done-yet-4966
branch_author__teratorn teratorn
status__closed closed
resolution__fixed fixed
component__web web
keywords__easy easy
time__1300585888000000 1300585888000000
changetime__1331687417000000 1331687417000000
version__None None
owner__teratorn teratorn
cc__jknight cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tenth10th's avatar" src="https://avatars.githubusercontent.com/u/854235?s=50" width="50" height="50">](https://github.com/tenth10th)<a name="note_3"></a>|@tenth10th commented|
|-|-|

It doesn't seem like PyDoctor wants to document NOT_DONE_YET, even if it is included in __all__, but that may be related to [this bug](https://bugs.launchpad.net/pydoctor/+bug/685376) wherein PyDoctor ignores constants?

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)|@teratorn set owner to @teratorn|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_7"></a>|@teratorn commented|
|-|-|

(In [33760]) Branching to 'not-done-yet-4966'

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_8"></a>|@teratorn set status to `closed`|
|-|-|

(In [33767]) Merge not-done-yet-4966: provide an __all__ in t.web.server.py to help out doc building

Author: necaris
Reviewer: teratorn
Fixes: [#4966](https://github.com/twisted/twisted/issues/4966)

**Comment by twisted-trac** [comment]
|[<img alt="teratorn's avatar" src="https://avatars.githubusercontent.com/u/154069?s=50" width="50" height="50">](https://github.com/teratorn)<a name="note_9"></a>|@teratorn commented|
|-|-|

Forgot to remove review keyword.
